docs/guides/server/configuration.adoc

```diff
@@ -101,6 +103,21 @@ You can also specify an explicit configuration file location using the `[-cf|--c
 
 <@kc.start rootParameters="--config-file=/path/to/myconfig.conf"/>
 
+=== Format for loading properties from a Java KeyStore
+
+Thanks to Keystore Configuration Source you can directly load properties from a Java KeyStore using the `[--config-keystore]` and `[--config-keystore-password]` options.
+Optionally, you can specify the KeyStore type using the `[--config-keystore-type]` option. By default, the KeyStore type is `PKCS12`.
+
+The secrets in a KeyStore needs to be stored using `PBE` (password-based encryption) key algorithm, where a key is derived from a KeyStore password.
```

Comment:
```suggestion
The secrets in a KeyStore need to be stored using the `PBE` (password-based encryption) key algorithm, where a key is derived from a KeyStore password.
```

Reply:
@andymunro thanks, I corrected the sentence.